modele-social/règles/salarié.yaml

```
@@ -416,7 +416,10 @@ contrat salarié . activité partielle . indemnités . base:
   formule:
     multiplication:
       assiette: retrait absence
-      taux: 70%
+      # Historique:
+      # - avant le 01/07/2021: 70%
+      # - à partir du 01/07/2021 : 60% (sauf pour les secteurs protégés, date repoussée au 01/09/2021)
+      taux [ref]: 70%
```

`[ref]` est déprécié, on prefère utiliser `nom`
```suggestion
      taux: 
        valeur: 70%
        nom: taux
```